pkg/api/perf_test.go

```diff
@@ -130,26 +130,26 @@ func (bs *benchState) cleanupBenchState() {
 	bs.deleteGRPCServer()
 }
 
-func (bs *benchState) Check(ctx context.Context, bag *attribute.MutableBag, output *attribute.MutableBag) rpc.Status {
+func (bs *benchState) Check(ctx context.Context, bag attribute.Bag, output *attribute.MutableBag) rpc.Status {
```

Comment:
MutableBag is a struct, whereas Bag is an interface. Don't want pointer to an interface in general in Go.